ui/src/app/views/error-handler/component.html

```diff
@@ -0,0 +1,4 @@
+<div *ngIf="this.notifyFiveHundred" class="error-handler">
```

Comment:
https://vmware.github.io/clarity/documentation/modals ?

My ideal UI element for this is a "toast" or "butterbar" (like GMail's "Failed to send. Next retry in 10s..." thing), but I don't see one ready in Clarity so I think this is a good place to start. (I like a toast because it doesn't block you from doing other things, or clicking the submit button again if you really want to.)

Reply:
+1 to a toast. -1 to modal dialogs. :)

Random Google search found this:

https://www.npmjs.com/package/angular2-toaster

(I'm not sure if we want to use it.)

Reply:
Also, beware that Clarity doesn't play nice with other libs. It tends to override much of the styling of other components.

Reply:
I looked for a toast yesterday, and it looked like clarity had one 'in queue'. I was planning on throwing a placeholder in and then keeping an eye on that. I can look at the angular2-toaster to see if that is reasonable.